src/store.rs

```diff
@@ -142,142 +172,5 @@ pub fn initial_decommit(storage: &mut dyn Storage, address: H160) -> Vec<U256> {
-        let key = RocksDBKey::HashToByteCode(hash);
-        let res = self
-            .db
-            .get(key.encode())
-            .map_err(|_| StorageError::KeyNotPresent)?;
-        Ok(res.map(|contract_code| contract_code.chunks_exact(32).map(U256::from).collect()))
-    }
-
-    fn add_contract(&mut self, hash: U256, code: Vec<U256>) -> Result<(), StorageError> {
-        let key = RocksDBKey::HashToByteCode(hash);
-        self.db
-            .put(key.encode(), encode_contract(code))
-            .map_err(|_| StorageError::ReadError)
-    }
-
-    fn storage_read(&self, key: StorageKey) -> Result<Option<U256>, StorageError> {
-        let key = RocksDBKey::ContractAddressValue(key.address, key.key);
-        let res = self
-            .db
-            .get(key.encode())
-            .map_err(|_| StorageError::ReadError)?;
-        match res {
-            Some(result) => {
-                let mut value = [0u8; 32];
-                value.copy_from_slice(&result);
-                Ok(Some(U256::from_big_endian(&value)))
-            }
-            None => Ok(None),
-        }
-    }
-
-    fn storage_write(&mut self, key: StorageKey, value: U256) -> Result<(), StorageError> {
-        let key = RocksDBKey::ContractAddressValue(key.address, key.key);
-        self.db
-            .put(key.encode(), encode(&value))
-            .map_err(|_| StorageError::WriteError)
-    }
-
-    fn storage_drop(&mut self, key: StorageKey) -> Result<(), StorageError> {
-        let key = RocksDBKey::ContractAddressValue(key.address, key.key);
-        self.db
-            .delete(key.encode())
-            .map_err(|_| StorageError::WriteError)
-    }
-
-    fn get_state_storage(&self) -> &HashMap<StorageKey, U256> {
-        unimplemented!()
-    }
-
-    fn get_all_keys(&self) -> Vec<StorageKey> {
-        let mut iter = self.db.raw_iterator();
-        iter.seek_to_first();
-        let mut keys = Vec::new();
-        while iter.valid() {
-            let key = iter.key().unwrap();
-            let address = H160::from_slice(&key[..20]);
-            let value = U256::from_big_endian(&key[20..]);
-            keys.push(StorageKey::new(address, value));
-            iter.next();
-        }
-        keys
-    }
-    fn fake_clone(&self) -> InMemory {
-        unimplemented!()
-    }
-}
-
-/// Encode a U256 into a byte vector to store and read from RocksDB.
-pub fn encode(value: &U256) -> Vec<u8> {
-    let mut encoded = Vec::new();
-    for key in value.0.iter().rev() {
-        let new_key = key.to_be_bytes().to_vec();
-        encoded.extend(new_key);
-    }
-    encoded
-}
-
-pub fn encode_contract(contract: Vec<U256>) -> Vec<u8> {
-    let mut encoded: Vec<u8> = vec![];
-    for word in contract {
-        let encoded_word = encode(&word);
-        encoded.extend_from_slice(&encoded_word);
-    }
-
-    encoded
+    contract_storage.decommit(code_key).unwrap().unwrap()
```

Comment:
I get that the idea here is that if the initial decommit fails, then the execution cannot continue, but I'd prefer to avoid unwraps here, so lets either:
A - Return a Result.
B - Replace unwrap with expect and an error description.

Although I would prefer option A since we don't forcefully panic on whoever calls this function.